### PR TITLE
Problem: default timeout for integration_tests is too long

### PR DESCRIPTION
**Pull request by mmsqe**
for more info: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepstimeout-minutes

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

**Review by yihuang — APPROVED** [review]

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@ concurrency:
 jobs:
   integration_tests:
     runs-on: ubuntu-latest
+    timeout-minutes: 180
```
Comment (thomas-nguy) [inline .github/workflows/test.yml:19]: mmm it is still quite long?

Maybe the default timeout for wait_for_fn could also be reduce

Comment (mmsqe) [reply .github/workflows/test.yml:19]: yes, for case like go [update](https://github.com/crypto-org-chain/cronos/pull/845)

Comment (mmsqe) [reply .github/workflows/test.yml:19]: the log seems not wait but just workflow stuck
```
2023-02-09T03:11:47.7203425Z test_basic.py::test_tx_inclusion[True-25000000] 2023-02-09 03:11:47,719 INFO waiting for cronos_777-1-node0 to stop
2023-02-09T03:11:47.7229433Z 2023-02-09 03:11:47,722 INFO stopped: cronos_777-1-node0 (terminated by SIGTERM)
2023-02-09T03:11:48.2776025Z 2023-02-09 03:11:48,277 INFO spawned: 'cronos_777-1-node0' with pid 24606
2023-02-09T03:11:48.2838310Z 2023-02-09 03:11:48,283 INFO stopped: cronos_777-1-node1 (terminated by SIGTERM)
2023-02-09T03:11:48.2877096Z 2023-02-09 03:11:48,287 INFO spawned: 'cronos_777-1-node1' with pid 24607
2023-02-09T03:11:48.2917123Z cronos_777-1-node0: stopped
2023-02-09T03:11:48.2917811Z cronos_777-1-node0: updated process group
2023-02-09T03:11:48.3055928Z cronos_777-1-node1: stopped
2023-02-09T03:11:48.3056308Z cronos_777-1-node1: updated process group
2023-02-09T03:11:48.3056606Z cronos_777-1-node2: stopped
2023-02-09T03:11:48.3056901Z cronos_777-1-node2: updated process group
2023-02-09T03:11:51.7649237Z 2023-02-09 03:11:51,764 INFO success: cronos_777-1-node0 entered RUNNING state, process has stayed up for > than 3 seconds (startsecs)
2023-02-09T03:11:51.7650298Z 2023-02-09 03:11:51,764 INFO success: cronos_777-1-node1 entered RUNNING state, process has stayed up for > than 3 seconds (startsecs)
2023-02-09T08:12:31.5353765Z ##[error]The operation was canceled.
```

Comment (thomas-nguy) [reply .github/workflows/test.yml:19]: I see, then should be ok.

I have seen a lot of random failure on the integration tests lately. May be worth to investigate the reason

**Pull request merged**